cmd/server.go

```diff
@@ -5,6 +5,7 @@ import (
 	"os/signal"
 	"parcel-service/internal/app/server"
 	"parcel-service/internal/pkg/postgres"
+	"parcel-service/internal/app/parcel_carrier"
```

Comment:
Please change the service name to carrier to avoid conflict, as it is a service of parcel no need to mention parcel_ individually.